packages/pangea-sdk/pangea/verify_audit.py

```diff
@@ -184,10 +215,9 @@ def _verify_signature(data: t.Dict) -> t.Optional[bool]:
             logger.debug("Obtaining signature and public key from the event")
             sign_envelope = create_signed_envelope(data["event"])
```

Comment:
Do we keep using dict for any particular reason? Don't we prefer to use data classes?

Reply:
In fact this tool works with different inputs: search responses and "verification artifacts" from the PUC, and it has also been released before pydantic refactor. Feel free to refactor if you consider